src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ListApprenticeshipsController.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using SFA.DAS.ApprenticeCommitments.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.Controllers
+{
+    [ApiController]
+    public class ListApprenticeshipsController : ControllerBase
+    {
+        private readonly IInternalApiClient<ApprenticeCommitmentsConfiguration> _client;
+
+        public ListApprenticeshipsController(IInternalApiClient<ApprenticeCommitmentsConfiguration> client)
+        {
+            this._client = client;
+        }
+
+        [HttpGet("/apprentices/{apprenticeId}/apprenticeships")]
+        public async Task<IActionResult> AddApprenticeship(Guid apprenticeId)
+        {
+            var response = await _client.Get<GetApprenticeshipsRepsonse>(
+                            new GetApprenticeshipsRequest(apprenticeId));
+            if (response == null)
+                return NotFound();
+            else
+                return Ok(response);
+        }
```

Comment:
One of the patterns people complain to me about is 'if/else' statements should be bracketed '{}' but I don't mind wither way. Not sure if it's really a ESFA standard or just a personal preference for those